test/onnx/test_fx_to_onnx_with_onnxruntime.py

```diff
@@ -636,6 +636,22 @@ def test_gpt2_tiny(self):
             model, [], inputs, additional_test_inputs=[((), another_inputs)]
         )
 
+    @pytorch_test_common.skip_min_ort_version(
```

Comment:
How long should we keep those? I feel we can delete but like to hear your thoughts @justinchuby @titaiwangms

Reply:
TBH, I think we are free to delete it. But I am not sure if there is any internal tests using older version of ort.....

Reply:
haha there shouldn't, let's do it separately then.